src/routes/formsgSiteCreation.ts

```diff
@@ -51,6 +52,7 @@ export class FormsgRouter {
     const requesterEmail = getField(responses, REQUESTER_EMAIL_FIELD)
     const siteName = getField(responses, SITE_NAME_FIELD)
     const repoName = getField(responses, REPO_NAME_FIELD)
+    const ownerEmail = getField(responses, OWNER_NAME_FIELD)?.toLowerCase()
```

Comment:
Nit: could we do a strip on the string? 
worried if ops put space then the string equality operation fails and it registers as new email 
(maybe this error case is captured downstream)

Reply:
good catch, added in https://github.com/isomerpages/isomercms-backend/pull/679/commits/80cfbbd63ade3c5d9dc78a2992b19c8edbf1a89a